### PR TITLE
Added test when files have final newline

### PR DESCRIPTION
**Pull request by grogy**
This solve problem on https://github.com/pehapkari/pehapkari.cz/pull/310#discussion_r134096432

### CODE REVIEW

**Comment by grogy** [comment]
I created check as test. The reason for it is existing of correct version `.editorconfig` (https://github.com/pehapkari/pehapkari.cz/blob/master/.editorconfig). It solve this problem automaticaly. Only test missing for final newlines.

File: bin/check_files_has_final_newline.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+find \
+    bin docs source/assets/css/ source/*.latte source/konference/ tests\
+    -type f -print0 \
+    | xargs -0 -L1 bash -c 'test "$(tail -c 1 "$0")" && echo "No new line at end of file $0" && exit 1'
```
Comment (TomasVotruba) [inline bin/check_files_has_final_newline.sh]: Thank you for the PR!

Is that fixing things or just finding it?
Could it also add a new line to those files?

Comment (grogy) [inline bin/check_files_has_final_newline.sh]: It only finding problems. Are you sure than you auto-repair missing newlines? I mean it is duplicate and magic, but the decision I let on you.

Comment (TomasVotruba) [inline bin/check_files_has_final_newline.sh]: The previous fixer does that, so does coding standard, so I would keep it common.

Comment (TomasVotruba) [inline bin/check_files_has_final_newline.sh]: You write this should resolve `.editorconfig` automatically? If so, maybe I have wrong configuration. In that case this all would be redundant.

Could you check that it fixes that in your PHPStorm please? Let me know the version.

**Comment by TomasVotruba** [comment]
I'm sorry about my over-engineered suggestion.

Resolved by https://github.com/pehapkari/pehapkari.cz/pull/310#discussion_r134756888

**Pull request closed without merging**